packages/ceramic-core/src/ceramic.ts

```diff
@@ -41,6 +41,7 @@ class Ceramic {
       '3id': new ThreeIdHandler(),
       'tile': new TileHandler(this),
       'account-link': new AccountLinkHandler(),
+      undefined: new ThreeIdHandler()
```

Comment:
This is temporary, would like to replace this object mapping with a function or something a bit smarter to allow for default values